project.janet

```diff
@@ -0,0 +1,15 @@
+(declare-project
+  :name "janetui"
+  :url "https://github.com/janet-lang/janetui"
```

Comment:
Please add a project description.

Reply:
with which keyword? :desc?

Reply:
I add a desc just for document purpose, as jpm doesn't consume it at all.